src/bin/pg_autoctl/pgsetup.c

```diff
@@ -1873,3 +1887,63 @@ pg_setup_standby_slot_supported(PostgresSetup *pgSetup, int logLevel)
 
 	return false;
 }
+
+
+/*
+ * pgsetup_parse_hba_level parses a string that represents an HBAEditLevel
+ * value.
+ */
+HBAEditLevel
+pgsetup_parse_hba_level(const char *level)
+{
+	HBAEditLevel enumArray[] = {
+		HBA_EDIT_SKIP,
+		HBA_EDIT_MINIMAL,
+		HBA_EDIT_APP
+	};
+
+	char *levelArray[] = { "skip", "minimal", "app", NULL };
+
+	for (int i = 0; levelArray[i] != NULL; i++)
+	{
+		if (strcmp(level, levelArray[i]) == 0)
+		{
+			return enumArray[i];
+		}
+	}
+
+	return HBA_EDIT_UNKNOWN;
+}
+
+
+/*
+ * pgsetup_hba_level_to_string returns the string representation of an
+ * hbaLevel enum value.
+ */
+char *
+pgsetup_hba_level_to_string(HBAEditLevel hbaLevel)
+{
+	switch (hbaLevel)
+	{
+		case HBA_EDIT_SKIP:
+		{
+			return "skip";
+		}
+
+		case HBA_EDIT_MINIMAL:
+		{
+			return "minimal";
+		}
+
+		case HBA_EDIT_APP:
+		{
+			return "app";
```

Comment:
maybe call it `"lan"` instead of `"app"` and use `HBA_EDIT_LAN` instead of `HBA_EDIT_APP`

Reply:
Yeah the naming there had me a little confused, as you can see. I am not fond on the name of the option `--pg-hba-lan` though it might work well enough, and might only get used in our docker-compose tests anyway.

Reply:
Renamed that way now.